mathesar/settings.py

```diff
@@ -1,8 +1,6 @@
 from django.conf import settings
 
-DEFAULTS = {
-    "APP_PREFIX": "mathesar_",
-}
+DEFAULTS = {}
```

Comment:
Remove this file, it's confusing to keep it around if we're not using it for anything.